teacher_module.rb

```diff
@@ -0,0 +1,17 @@
+module Teachers
```

Comment:
Only use modules when necessary -- for things that don't have a common parent that makes sense.  ApprenticeTeachers and Teachers are both `Staff` or `Instructors` -- make that the common superclass.